### DISCUSSION TITLE
vulnerabilityAlerts example results in a Configuration Error

### DISCUSSION THREAD

**Discussion by testworksau** [renovatebot/renovate#23399] [Request Help]
### How are you running Renovate?

Self-hosted Renovate

### If you're self-hosting Renovate, tell us what version of Renovate you run.

36.8.11

### If you're self-hosting Renovate, select which platform you are using.

github.com

### Was this something which used to work for you, and then stopped?

I am trying to get this working for the first time

### Describe the problem

According to [the documentation](https://docs.renovatebot.com/configuration-options/#vulnerabilityalerts), a correct configuration for Vulnerability alerts starts with:

```
{
  "groupName": null,
  "schedule": [],
  "dependencyDashboardApproval": false,
  "minimumReleaseAge": null,
  "rangeStrategy": "update-lockfile",
  "commitMessageSuffix": "[SECURITY]",
  "branchTopic": "{{{datasource}}}-{{{depName}}}-vulnerability",
  "prCreation": "immediate"
}
```

However, applying that configuration yields a Configuration Error:
```
INFO: Repository has invalid config (repository=ourorg/our-repo)
       "error": {
         "validationSource": ".github/renovate.json5",
         "validationError": "The renovate configuration file contains some invalid settings",
         "validationMessage": "packageRules[1]: Each packageRule must contain at least one match* or exclude* selector. Rule: {\"groupName\":null,\"schedule\":[],\"dependencyDashboardApproval\":false,\"minimumReleaseAge\":null,\"rangeStrategy\":\"update-lockfile\",\"commitMessageSuffix\":\"[SECURITY]\",\"branchTopic\":\"{{{datasource}}}-{{{depName}}}-vulnerability\",\"prCreation\":\"immediate\"}",
         "message": "config-validation",
         "stack": "Error: config-validation\n    at mergeRenovateConfig (/opt/containerbase/tools/renovate/36.8.11/node_modules/renovate/lib/workers/repository/init/merge.ts:232:19)\n    at getRepoConfig (/opt/containerbase/tools/renovate/36.8.11/node_modules/renovate/lib/workers/repository/init/config.ts:12:12)\n    at initRepo (/opt/containerbase/tools/renovate/36.8.11/node_modules/renovate/lib/workers/repository/init/index.ts:44:12)\n    at Object.renovateRepository (/opt/containerbase/tools/renovate/36.8.11/node_modules/renovate/lib/workers/repository/index.ts:54:14)\n    at attributes.repository (/opt/containerbase/tools/renovate/36.8.11/node_modules/renovate/lib/workers/global/index.ts:184:11)\n    at start (/opt/containerbase/tools/renovate/36.8.11/node_modules/renovate/lib/workers/global/index.ts:169:7)\n    at /opt/containerbase/tools/renovate/36.8.11/node_modules/renovate/lib/renovate.ts:18:22"
       }
```

I assume I just need to add in the following to the example, but wanted to raise in case you weren't already aware.

```
      "matchPackagePatterns": ["*"],
```

### Have you created a minimal reproduction repository?

I have explained in the description why a minimal reproduction is impossible

**Comment by rarkins** [comment]
The documentation doesn't say anywhere that it's a package rule, and it's not

**Reply by testworksau** [reply]
Oh, thanks - I wonder why I thought that was the case 🤦🏼 

Just to confirm - is this enabled by default? ie. do we only need to provide config for vulnerabilityAlerts if we want to override the defaults, or do something like enable automerge?

```
  "vulnerabilityAlerts": {
    "automerge": true
}
```

**Reply by rarkins** [reply]
Enabled by default if you have the alerts enabled in GitHub and have granted access. Search debug logs for the string "vulnerability"
